Sources/TokamakStaticHTML/Shapes/_ShapeView.swift

```diff
@@ -183,3 +223,54 @@ extension _ShapeView: _HTMLPrimitive {
     }
   }
 }
+
+@_spi(TokamakStaticHTML) extension _ShapeView: HTMLConvertible {
```

Comment:
nit: let's keep declaration attributes consistently on a separate line
```suggestion
@_spi(TokamakStaticHTML) 
extension _ShapeView: HTMLConvertible {
```